### PR TITLE
Test CellCoordinate and FacetCoordinate

### PR DESCRIPTION
**Pull request by miklos1**
Requires firedrakeproject/tsfc#36.

### CODE REVIEW

File: tests/regression/test_cellcoordinate.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+def test_facet_coordinate_dx(mesh):
+    with pytest.raises(UFLException):
+        assemble(FacetCoordinate(mesh)[0]*dx)
+
+
+def test_facet_coordinate_ds(mesh):
+    assert np.allclose(0.5 * (2 + sqrt(2)), assemble(FacetCoordinate(mesh)[0]*ds))
+
```
Comment (wence-) [inline tests/regression/test_cellcoordinate.py]: Restrictions?

**Comment by wence-** [comment]
Otherwise fine.

File: tests/regression/test_cellcoordinate.py (hunk 1 above)
Comment (miklos1) [inline tests/regression/test_cellcoordinate.py]: Should work, but yes, I should add a test.

Comment (miklos1) [inline tests/regression/test_cellcoordinate.py]: Done.

**Pull request merged**